apps/src/studio/collisionMaskWalls.js

```diff
@@ -100,4 +107,36 @@ export default class CollisionMaskWalls extends Walls {
     return arr;
   }
 
+  /**
+   * Construct an image data URI from the wallData that shows the walls in a
+   * solid color.
+   */
+  wallOverlayURI(imageData, wallMap, color = WALL_COLOR) {
+    const data = imageData.data;
+    for (let y = 0; y < this.height; y++) {
+      for (let x = 0; x < this.width; x += BITS_PER_BYTE) {
+        const currentByte = wallMap[(y * this.bytesPerRow) + (x / BITS_PER_BYTE)];
+        for (let k = 0; k < BITS_PER_BYTE; k++) {
+          const map = 1 << k;
+          const imageDataIndex = ((y * this.width) + x + k) * BYTES_PER_PIXEL;
+          if (currentByte & map) {
+            // Wall pixel, set color
+            data[imageDataIndex + 0] = color.R;
+            data[imageDataIndex + 1] = color.G;
+            data[imageDataIndex + 2] = color.B;
+            data[imageDataIndex + 3] = 255;
```

Comment:
nit: could more simply do this with `data.splice(imageDataIndex, 4, color.R, color.G, color.B, 255)`